dbt_project.yml

```diff
@@ -24,20 +24,20 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "dbt_packages"
 
 vars:
-  conversion_window_start_date: '' # conversion window start date
-  conversion_window_end_date: '' # conversion window end date is dynamically set to current_date()-1 in the code
-  conversion_hosts: ['a.com'] # url_hosts to consider
-  path_transforms: {'exposure_path': null} # dictionary of path transforms (and their argument, null if none) to perform on the full conversion path (see udfs.sql)
-  path_lookback_steps: 0 # Limit for the number of marketing channels to look at before the conversion (0 = unlimited)
-  path_lookback_days: 30 # Restrict to marketing channels within this many days of the conversion (recommended: 30 / 14 / 7)
+  snowplow__conversion_window_start_date: '' # conversion window start date
+  snowplow__conversion_window_end_date: '' # conversion window end date is dynamically set to current_date()-1 in the code
+  snowplow__conversion_hosts: ['a.com'] # url_hosts to consider
+  snowplow__path_transforms: {'exposure_path': null} # dictionary of path transforms (and their argument, null if none) to perform on the full conversion path (see udfs.sql)
+  snowplow__path_lookback_steps: 0 # Limit for the number of marketing channels to look at before the conversion (0 = unlimited)
+  snowplow__path_lookback_days: 30 # Restrict to marketing channels within this many days of the conversion (recommended: 30 / 14 / 7)
 
   consider_intrasession_channels: false # false = only considers the channel at the start (first page view) of the session, true = consider multiple channels in the conversion session as well as historical channels
```

Comment:
What's the reason for not prefixing this one?

Reply:
Nothing apart from being in a hurry. Will change it!